### PR TITLE
RHOARDOC-1669 add missing instruction for installing on OpenShift

### PR DESCRIPTION
**Pull request by chrira**
Installation does not work without this customization of the custom resource.

### CODE REVIEW

**Comment by chrira** [comment]
> @chrira Thanks for the feedback! I have a few comments on the wording. Let me know what you think.

I do not find your comments. Where do I have to look at them?

**Comment by huntley** [comment]
Now I don't see them either. I will recreate.

File: docs/topics/proc_installing-launcher-operator.adoc [hunk 1]
```diff
@@ -126,7 +126,7 @@ Based on the templates, OpenShift automatically installs:
 * an access role definition for the Launcher operator
 * access role bindings for the Launcher operator
 
-. Install the `launcher_cr.yaml` custom resource to initiate the deployment of the {name-launcher} tool.
+. Customize the `launcher_cr.yaml` custom resource with your OpenShift Console URL and install it in order to initiate the deployment of the {name-launcher} tool.
```
Comment (huntley) [inline docs/topics/proc_installing-launcher-operator.adoc]: I think this works better as two separate steps: modify the custom resource and then install it. Something like:

8. Edit the 'launcher_cr.yml' custom resource to add your OpenShift Console URL:

####### OpenShift Configuration
  openshift:
    consoleUrl: #<the OpenShift Console Url (ex: 'https://192.168.64.4:8443')>#

9. Install the 'launcher_cy.yaml' custom resource to deploy the {name-launcher} tool.

**Comment by huntley** [comment]
@chrira Ok. Sorry about that. Comments are back. I have also asked one of our QE folks to take a look as well.

File: docs/topics/proc_installing-launcher-operator.adoc (hunk 1 above)
Comment (dsimansk) [reply docs/topics/proc_installing-launcher-operator.adoc]: I do agree with Huntley here. I'd like a bit more two steps approach with an example snippet.

**Comment by chrira** [comment]
Thank you for the review. I changed the file according to your input.
I found a note on updating the launcher_cr.yaml ahead in the document. But it was before the download instruction of the ZIP file, which includes the launcher_cr.yaml file.
I moved that part.

**Comment by huntley** [comment]
[test]

**Comment by inoxx03** [comment]
[test]

**Pull request merged**